test/e2e/util.go

```diff
@@ -55,7 +55,6 @@ const (
 
 	olmConfigMap = "olm-operators" // No-longer used, how long do we keep this around?
 
-	// sync name with scripts/install_local.sh
```

Comment:
scripts/install_local.sh does not seem to reference anything to with the packageserver...